data/projects.js

```diff
@@ -501,5 +501,28 @@ coala should support generating metrics for your code.",
 		"tags" : ["Bears", "cib"],
 		"markdown": "improve_installation.md",
 		"collaborating_projects" : ["coala"]
+	},
+	{
+	  "name" : "Enhancing coala-quickstart",
+	  "desc" : "Utilizing information from project's configurations and improving the user-experience.",
```

Comment:
I'd rather term this as `Extract useful data from a project's configuration files to build a relevant coafile.` as it's more specific.